rotkehlchen/assets/asset.py

```diff
@@ -916,6 +918,23 @@ def to_dict(self) -> Dict[str, Any]:
                 'underlying_tokens': underlying,
             }
 
+        if self.is_custom_asset():
+            custom_asset = CustomAsset.from_identifier(self.identifier)
+            asset_dict.pop('cryptocompare')
```

Comment:
I would say that this is not the best approach. I think we should better don't add these fields to the dict at all instead of adding and then removing them